dags/ricochet_dca.py

```diff
@@ -48,26 +46,10 @@
     dag=dag,
 )
 
-approve = ERC20ApprovalOperator(
-    task_id="approve",
-    web3_conn_id="infura",
-    ethereum_wallet=SWAPPER_WALLET_ADDRESS,
-    gas_multiplier=1.2,
-    gas=3000000,
-    max_gas_price=MAX_GAS_PRICE,
-    contract_address="0x2791Bca1f2de4661ED88A30C99A7a9449Aa84174",
-    spender="0x1b02dA8Cb0d097eB8D57A175b88c7D8b47997506",
-    amount=SWAP_AMOUNT,
-    dag=dag
-)
-
-confirm_approve = EthereumTransactionConfirmationSensor(
-    task_id="confirm_approve",
-    web3_conn_id="infura",
-    transaction_hash="{{task_instance.xcom_pull(task_ids='approve')}}",
-    confirmations=1,
-    poke_interval=5,
-    timeout=60 * 20,
+stall = RandomSleepOperator(
+    task_id="stall",
+    min=60*3,
+    max=60*15,
```

Comment:
Can we add a variable these?, also these are contents so please use the `class` from https://github.com/Ricochet-Exchange/ricochet-keeper/pull/29